CRITICAL - 2024-06-03 06:50:33 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "welcome_message.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('welcome_message.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('welcome_message', [], true)
 3 APPPATH\Controllers\Home.php(9): view('welcome_message')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-03 06:51:30 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "welcome_message.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('welcome_message.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('welcome_message', [], true)
 3 APPPATH\Controllers\Home.php(9): view('welcome_message')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-03 06:51:33 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "welcome_message.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('welcome_message.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('welcome_message', [], true)
 3 APPPATH\Controllers\Home.php(9): view('welcome_message')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-03 07:29:07 --> mysqli_sql_exception: Unknown column 'ne' in 'field list' in C:\xampp\htdocs\intern\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\intern\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `in...', 0)
#1 C:\xampp\htdocs\intern\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `in...')
#2 C:\xampp\htdocs\intern\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `in...')
#3 C:\xampp\htdocs\intern\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `in...', Array, false)
#4 C:\xampp\htdocs\intern\app\Controllers\Home.php(28): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\intern\system\CodeIgniter.php(933): App\Controllers\Home->addData()
#6 C:\xampp\htdocs\intern\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\intern\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\intern\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\intern\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\intern\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-06-03 07:29:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'ne' in 'field list'
[Method: POST, Route: Home/addData]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `interntable` (`ne`, `mobile_number`, `email`, `created_on`) VALUES (:ne:, :mobile_number:, :email:, :created_on:)', [...], false)
 2 APPPATH\Controllers\Home.php(28): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-03 07:29:07 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'ne' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `interntable` (`ne`, `mobile_number`, `email`, `created_on`) VALUES (\'rere\', \'2325235\', \'abc@gmail.com\', \'2024-06-03 07:29:07\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `interntable` (`ne`, `mobile_number`, `email`, `created_on`) VALUES (\'rere\', \'2325235\', \'abc@gmail.com\', \'2024-06-03 07:29:07\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `interntable` (`ne`, `mobile_number`, `email`, `created_on`) VALUES (:ne:, :mobile_number:, :email:, :created_on:)', [...], false)
 4 APPPATH\Controllers\Home.php(28): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-03 07:29:07 --> [Caused by] mysqli_sql_exception: Unknown column 'ne' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `interntable` (`ne`, `mobile_number`, `email`, `created_on`) VALUES (\'rere\', \'2325235\', \'abc@gmail.com\', \'2024-06-03 07:29:07\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `interntable` (`ne`, `mobile_number`, `email`, `created_on`) VALUES (\'rere\', \'2325235\', \'abc@gmail.com\', \'2024-06-03 07:29:07\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `interntable` (`ne`, `mobile_number`, `email`, `created_on`) VALUES (\'rere\', \'2325235\', \'abc@gmail.com\', \'2024-06-03 07:29:07\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `interntable` (`ne`, `mobile_number`, `email`, `created_on`) VALUES (:ne:, :mobile_number:, :email:, :created_on:)', [...], false)
 5 APPPATH\Controllers\Home.php(28): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
